rosbag2_cpp/src/rosbag2_cpp/cache/circular_message_cache.cpp

```diff
@@ -38,7 +39,8 @@ void CircularMessageCache::push(std::shared_ptr<const rosbag2_storage::Serialize
   primary_buffer_->push(msg);
 }
 
-std::shared_ptr<MessageCacheCircularBuffer> CircularMessageCache::consumer_buffer()
+std::shared_ptr<rosbag2_cpp::cache_interfaces::BaseCacheBufferInterface> CircularMessageCache::
```

Comment:
Perhaps it could be shortened as:
```suggestion
std::shared_ptr<rCacheBufferInterface> CircularMessageCache::
```